spec/index.html

```diff
@@ -718,7 +718,8 @@ <h2>Literals</h2>
         non-empty <dfn>language tag</dfn> as defined by [[!BCP47]]. The
         language tag MUST be well-formed according to
         <a data-cite="bcp47#section-2.2.9">section 2.2.9</a>
-        of [[!BCP47]].</li>
+        of [[!BCP47]],
+        and MUST be case normalized consistently (e.g., to lower case)..</li>
```

Comment:
```suggestion
        and MUST be case normalized consistently (e.g., to lower case).</li>
```

Reply:
Note that line 727 should end with a comma, not a period.

Reply:
@TallTed All the points are inconsistent. Either the second le should and in a '.', or they all should end with a ',' (other than the last). I leave it to your discretion.